### PR TITLE
Diff: Increment 2nd paramter of range in Diff.delete_lines

### PR DESCRIPTION
**Pull request by kmo1**
Closes https://github.com/coala-analyzer/coala/issues/2641

### CODE REVIEW

**Comment by Asnelchristian** [comment]
what about ``Diff: Increase `line_nr_end` in `delete_lines` `` as your short log

**Comment by sils1297** [comment]
I think it's good after the commit message was changed :)

File: coalib/results/Diff.py [hunk 1]
```diff
@@ -358,7 +358,7 @@ def delete_lines(self, line_nr_start, line_nr_end):
         """
         Delete lines in a specified range.
         """
-        for line_nr in range(line_nr_start, line_nr_end):
+        for line_nr in range(line_nr_start, line_nr_end + 1):
```
Comment (Makman2) [inline coalib/results/Diff.py]: please add some documentation that the given end-line is deleted inclusively :+1:

**Comment by Makman2** [comment]
unack 28a4631

File: coalib/results/Diff.py (hunk 1 above)
Comment (kmo1) [inline coalib/results/Diff.py]: @Makman2 - Should I just be updating the comments or is there documentation somewhere specific that should be updated?

Comment (Makman2) [inline coalib/results/Diff.py]: right above is the docstring, a little mention there would be good :)

**Comment by Makman2** [comment]
ack 28a4631 e2ece4e

**Comment by Makman2** [comment]
I thought of fixing the docstring in the previous commit, but in fact it's alright, as before it wasn't really well documented either :+1: :)

**Comment by Makman2** [comment]
all that's missing now is a rebase, then we can merge

**Pull request closed without merging**